QEfficient/generation/text_generation_inference.py

```diff
@@ -157,8 +158,8 @@ def cloud_ai_100_exec_kv_helper(
     # Skip inputs/outputs
     session.skip_buffers([x for x in session.input_names + session.output_names if x.startswith("past_")])
 
-    # Read prompt and ctx len from session
-    batch_size, _, ctx_len, _ = session.bindings[session.binding_index_map["past_key.0"]].dims
+    # Read prompt and batch_size from session
+    batch_size, _ = session.bindings[session.binding_index_map["input_ids"]].dims
```

Comment:
This may fail to get proper batch_size, since input_ids is specialized, we use the allowed_shapes instead as mentioned below for prefill_seq_len. Will try to fix this in a new commit.